src/components/ScrollToTopButton/ScrollToTopButton.module.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../assets/styles/theme';
+
+.scrollToButtonLight {
```

Comment:
nei ma sensu powiela styli do dwóch klas, zrób to tak, żę w jednej masz style a w motywie czarnym masz tylko filtr kolorów https://github.com/UEK-i-team/documentation/blob/main/best-practices-in-gatsby.MD#51-u%C5%BCywanie-motywu-jasnego-i-ciemnego